README.md

```diff
@@ -305,12 +305,9 @@ $ drive pull --files a1/b2
 $ drive pull --directories tf1
 ```
 
-## Note: Checksum verification:
+Due to popular demand, by default checksum verification is turned off. It was deemed to be quite vigorous and unnecessary for most cases, in which size + modTime differences are sufficient to detect file changes. The discussion stemmed from issue [#117](https://github.com/odeke-em/drive/issues/117).
```

Comment:
I removed the note because it seemed strange to have a `##` header as a sub-header of a `###` header. But I guess it's a good way to get direct people's attention. Although now that checksum verification is turned off by default, it may not be as important anymore.

Anyway, I added back the note. My pull request was mainly meant because of the "Pushing" chapter, which confused me about whether checksumming was on or off by default.

Also added the comma.